src/views/onboarding/login/index.tsx

```diff
@@ -14,55 +13,78 @@ import {
 import { OnboardingTranslation } from "../context/onboardingTranslation";
 
 import "./styles.scss";
+import { withRouter } from "react-router-dom";
+import { homeRoute } from "../../../shared/routes/routes";
 
-const Login = () => {
+const Login = ({ history }) => {
   const { t: translate } = useTranslation();
 
   // @ts-ignore
-  const { handleChange, values, errors } = useFormik({
+  const { handleChange, values, errors, dirty, isValid } = useFormik({
     initialValues: defaultLoginValues,
     validationSchema: loginValidators,
   });
 
   const submitLogin = async (values) => {
     try {
       console.log(values);
-      await loginRequest(values);
+      const { data } = await loginRequest(values);
+      localStorage.setItem("jwtToken", data["accessToken"]);
+      history.push(homeRoute());
     } catch (error) {
       // TODO: Handle error https://brick-link.atlassian.net/browse/BRIC-15
     }
   };
 
-  console.log(errors)
-
   return (
-    <Container className={"bl-login__container"}>
-      <label>{translate(OnboardingTranslation.loginPlaceholder)}</label>
-      <Row>
-        <Col xs={4} lg={12}>
-          <Input
-            id={LoginFormFields.email}
-            placeholder={"test"}
-            name={LoginFormFields.email}
-            onChange={handleChange}
-            value={values.email}
-            label={'nemanja'}
-          />
-          <Input
-            id={LoginFormFields.password}
-            placeholder={"test"}
-            name={LoginFormFields.password}
-            onChange={handleChange}
-            value={values.password}
-            label={'nemanja'}
+    <div className={"p-login"}>
+      <Container
+        className={"p-login__container p-flex p-flex-column p-items-center"}
+      >
+        <div className={"p-login__logo"}>LOGO</div>
+        <label className={"p-login__label"}>
+          <h1>{translate(OnboardingTranslation.signInPlaceholder)}</h1>
+        </label>
+        <Row className={"p-login__input"}>
+          <Col xs={4} lg={12}>
+            <Input
+              id={LoginFormFields.email}
+              placeholder={translate(OnboardingTranslation.emailPlaceholder)}
+              name={LoginFormFields.email}
+              onChange={handleChange}
+              value={values.email}
+              errors={errors}
+              type={"email"}
+            />
+            <Input
+              id={LoginFormFields.password}
+              placeholder={translate(OnboardingTranslation.passwordPlaceholder)}
+              name={LoginFormFields.password}
+              onChange={handleChange}
+              value={values.password}
+              errors={errors}
+              type={"password"}
+            />
+          </Col>
+          <div className={"p-login__forgot"}>Forgot password?</div>
+        </Row>
+        <Row>
+          <Button
+            label={translate(OnboardingTranslation.signInPlaceholder)}
+            onClick={() => submitLogin(values)}
```

Comment:
No need for passing values since they are component props, we can access them inside fucntion

Reply:
So this can be
`onClick={submitLogin}`